docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -169,4 +432,112 @@ TBD
+
+.. code:: sh
+
+  values:
+    greeting: Hello
+    recipient: World
+
+* Validate a YAML config against a given JSON schema:
+
+  .. code:: sh
+
+    $ uw config validate --schema-file schema.jsonschema --input-file values.yaml
+    [2024-01-03T17:23:07]     INFO 0 schema-validation errors found
+
+  Shell redirection via ``|``, ``>``, et al may also be used to stream output to a file, another process, etc.
+
+
+* Read the config from ``stdin`` and print validation results to ``stdout``:
+
+  .. code:: sh
+
+    $ cat values.yaml | uw config validate --schema-file schema.jsonschema
+    [2024-01-03T17:26:29]     INFO 0 schema-validation errors found
+
+
+* However, reading the schema from ``stdin`` is not supported:
+
+  .. code:: sh
+
+    $ cat schema.jsonschema | uw config validate -input-file values.yaml
+    uw config validate: error: the following arguments are required: --schema-file
+
+* If a config fails validation, differences from the schema will be displayed. For example, with ``recipient: World`` removed from ``values.yaml``:
+
+  .. code:: sh
+
+    $ uw config validate --schema-file schema.jsonschema --input-file values.yaml
+    [2024-01-03T17:31:19]    ERROR 1 schema-validation error found
+    [2024-01-03T17:31:19]    ERROR 'recipient' is a required property
+    [2024-01-03T17:31:19]    ERROR 
+    [2024-01-03T17:31:19]    ERROR Failed validating 'required' in schema['properties']['values']:
+    [2024-01-03T17:31:19]    ERROR     {'additionalProperties': False,
+    [2024-01-03T17:31:19]    ERROR      'properties': {'greeting': {'type': 'string'},
+    [2024-01-03T17:31:19]    ERROR                     'recipient': {'type': 'string'}},
+    [2024-01-03T17:31:19]    ERROR      'required': ['greeting', 'recipient'],
+    [2024-01-03T17:31:19]    ERROR      'type': 'object'}
+    [2024-01-03T17:31:19]    ERROR 
+    [2024-01-03T17:31:19]    ERROR On instance['values']:
+    [2024-01-03T17:31:19]    ERROR     {'greeting': 'Hello'}
+
+* Request verbose log output:
+
+  .. code:: sh
+
+    $ uw config validate --schema-file schema.jsonschema --input-file values.yaml --verbose
+    [2024-01-03T17:29:46]    DEBUG Command: uw config validate --schema-file schema.jsonschema --input-file values.yaml --verbose
+    [2024-01-03T17:29:46]    DEBUG Dereferencing, initial value: {'values': {'greeting': 'Hello', 'recipient': 'World'}}
+    [2024-01-03T17:29:46]    DEBUG Rendering: {'values': {'greeting': 'Hello', 'recipient': 'World'}}
+    [2024-01-03T17:29:46]    DEBUG Rendering: {'greeting': 'Hello', 'recipient': 'World'}
+    [2024-01-03T17:29:46]    DEBUG Rendering: Hello
+    [2024-01-03T17:29:46]    DEBUG Rendering: World
+    [2024-01-03T17:29:46]    DEBUG Dereferencing, final value: {'values': {'greeting': 'Hello', 'recipient': 'World'}}
+    [2024-01-03T17:29:46]     INFO 0 schema-validation errors found
+
+  Note that ``uw`` logs to ``stderr`` and writes non-log output to ``stdout``, so the streams can be redirected separately:
```

Comment:
This may not be relevant since `uw config validate` doesn't really print anything to `stdout`.